### PR TITLE
[IMP] project: improve show revenues for archived products

### PR DESCRIPTION
**Pull request by rupa-odoo**
Before these commit if a product sold in an SO that created a project is archived , their revenues will no longer appear in the project.update even if the project is ongoing or the hours/units have been invoiced.

Improvements:
Revenues of the archived product will be visible.

task-3422547

### CODE REVIEW

**Review by MissingNoShiny — COMMENTED** [review]
LGTM, thanks for your work :slightly_smiling_face:

**Review by auon-odoo — APPROVED** [review]
Thanks!

**Review by xavierbol — COMMENTED** [review]
LGTM, thanks for your work. 🙂 

May I ask you to add a test? Thanks in advance! 🙂

**Comment by xavierbol** [comment]
Do you have any issue to add a test for that? 🧐

**Comment by rupa-odoo** [comment]
> Do you have any issue to add a test for that? 🧐

No, I have started working on a test. 
I was previously occupied with another task.

**Comment by rupa-odoo** [comment]
> Do you have any issue to add a test for that? 🧐

Hello @xavierbol ,
I have add a test case.
Thank you

File: addons/sale_project/tests/test_project_profitability.py [hunk 1]
```diff
@@ -1157,3 +1157,63 @@ def test_bills_without_purchase_order_are_accounted_in_profitability_sale_projec
...
+            'product_uom_qty':10,
+            'order_id':sale_order_revenue.id,
+        })
+
+        sequence_per_invoice_type = self.project._get_profitability_sequence_per_invoice_type()
+        profitability_item_data = {
+            'revenues': {
+                'data': [{
+                    'id': 'billable_fixed',
+                    'sequence': sequence_per_invoice_type['billable_fixed'],
```
Comment (xavierbol) [inline addons/sale_project/tests/test_project_profitability.py:1194]: billable_fixed is not found when we start this test only with `sale_project` module.

Can you check?

Comment (rupa-odoo) [reply addons/sale_project/tests/test_project_profitability.py]: > billable_fixed is not found when we start this test only with `sale_project` module.
> 
> Can you check?

Hello @xavierbol ,
I check that,
I moved test case to sale_timesheet module.

**Review by xavierbol — APPROVED** [review]
LGTM, thanks for your work. 🙂 

robodoo r+

**Pull request closed without merging**